### ISSUE TITLE
current gcc:11 image is broken

### ISSUE THREAD

**Issue by punytroll** [docker-library/gcc#96]
Similar to #95, when running the preinstalled program `convert` I get a this error:
```
root@888289dec9c0:/# convert
convert: /usr/local/lib64/libstdc++.so.6: version `GLIBCXX_3.4.30' not found (required by /lib/x86_64-linux-gnu/libicuuc.so.72)
```

**Comment by yosifkit** [comment]
😢 it looks like https://github.com/docker-library/gcc/pull/94 was not enough. We'll have to move gcc 11 back as well (PR incoming).  You can use `gcc:11-bullseye` to stick to the working image (and it'll continue to be updated since that'll be the newest Debian we could possibly use for GCC `11`).

**Comment by jordy-van-appeven-accerion** [comment]
Same problem seems to be the case for `gcc:9.5.0` 
```
root@e23d11f2fe83:/# apt update
apt: /usr/local/lib64/libstdc++.so.6: version `GLIBCXX_3.4.29' not found (required by /lib/x86_64-linux-gnu/libapt-private.so.0.0)
apt: /usr/local/lib64/libstdc++.so.6: version `GLIBCXX_3.4.29' not found (required by /lib/x86_64-linux-gnu/libapt-pkg.so.6.0)
```

**Comment by yosifkit** [comment]
@jordy-van-appeven-accerion, GCC 9 is fixed in #94, but the images have not been built and pushed to Docker Hub. You can switch to the `9-bullseye` or `9.5-bullseye` tags to get back to the old image (and that tag will be updated since that will be the supported Debian release for GCC 9).

See the [readme](https://github.com/docker-library/gcc/tree/4c74f6f7568101ac1cba565c0b009ed05dba8d8f#see-a-change-merged-here-that-doesnt-show-up-on-docker-hub-yet) -> https://github.com/docker-library/official-images/pull/14896
